docs/content/en/docs/Concepts/Falco/_index.md

```diff
@@ -65,3 +65,20 @@ The following commands add the FalcoSideKick chart and then installs with the pa
 - Make sure to change the URL value to point to your M9sweeper instance as well as enter the CLUSTER_ID of whatever cluster it is supposed to save to.
 
 - We recommend using the EBPF driver, however, if you have issues please refer to to the [installation page](https://falco.org/docs/getting-started/installation/#install-driver). Or you may try to use the kernel driver and set "falco.driver.kind=module" above.
+
+### Anomaly Detaction:
+- Anomaly instrusion detection plays vital role in protecting networks against malicious activities.
+- It detects unusual behaviors or threats in cloud-native environments with about 100 out-of-the-box security rules.
```

Comment:
Lets mention here that we will notify administrators of anomolies through email

Reply:
It's mentioned on line 81. Do you want me to move it up to line 72?

Reply:
Yeah, maybe we would put the two bullet points from 81 and 82 up under line 71, since they are related to anomoly detection and not the example of a violation.